css/styles.scss

```diff
@@ -3745,6 +3745,8 @@ a.copyright {
       clear: both;
       padding-top: 8px;
       border-top: 1px dashed #D2D2D2;
+      display: flex;
+      flex-flow: row wrap;
```

Comment:
this property broke right displayed items (not related to this pr, but i think adding right items is currently broken in 9.5/bugfixes, will check this afternoon).

You should change add to h_item a specific rule like this i think:
```scss
   .h_item {
      ...

      &.right {
         flex-direction: row-reverse;
      }
   }
```

Also, moving to flex layout seems to have sub divs to have 100% height of their container, don't know why

![image](https://user-images.githubusercontent.com/418844/81549344-dfe96a00-937e-11ea-8f9b-d0c7518a0e6e.png)

Reply:
at least for the 100% height, i think we cant to push status bar to the bottom

Reply:
nevermind my comment on adding right item, bug was on my side only

Reply:
> nevermind my comment on adding right item, bug was on my side only

What should I change then?

Reply:
you should adding the `&.right` rule

i was taking about 
> (not related to this pr, but i think adding right items is currently broken in 9.5/bugfixes, will check this afternoon)

Reply:
Ah. I see the issue now. Added that CSS. Thanks!